README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
```

Comment:
Two points here:

1. "installation" is misspelled as "instllation"
2. I think adding to PATH is part of installation so should be in that section. However, a paragraph linking back to installation instructions and reminder that `corral` may not be on the PATH seems fitting here.